resources/public/api/conf/1.0/lossClaims_list.raml

```diff
@@ -10,9 +10,11 @@ is:
   - errors.formatSelfEmploymentId
   - errors.clientOrAgentNotAuthorised
   - errors.notFoundNino
+  - errors.formatClaimType
```

Comment:
Can this be moved up with the other 400 errors (below `errors.formatSelfEmploymentId`)?